templates/CRM/common/deferredFinancialType.tpl

```diff
@@ -15,7 +15,7 @@ CRM.$(function($) {
   var more = $('.crm-button.validate').click(function(e) {
     var message = "{/literal} {if $context eq 'Event'}
         {ts escape='js'}Note: Revenue for this event registration will not be deferred as the financial type does not have a deferred revenue account setup for it. If you want the revenue to be deferred, please select a different Financial Type with a Deferred Revenue account setup for it, or setup a Deferred Revenue account for this Financial Type.{/ts}
-      {else if $context eq 'MembershipType'}
+      {elseif $context eq 'MembershipType'}
```

Comment:
In Smarty 4/5, `{else if}` works like `{elseif}`. But in Smarty 2, `{else if}`raises errors.